### PR TITLE
charts_12 Added Andi Chart

### PR DESCRIPTION
**Pull request by LtChae**
co-authored-by: Richard Jones <rajones@hntb.com>

### CODE REVIEW

File: andi/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: REST API to allow for dataset definition ingestion
+name: andi
+version: 1.0.0
```
Comment (ManApart) [inline andi/Chart.yaml]: We may want to inc this version.

File: docs/index.yaml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+  andi:
+  - apiVersion: v1
+    appVersion: "1.0"
+    created: 2019-07-16T14:41:51.46638071-04:00
+    description: REST API to allow for dataset definition ingestion
+    digest: 87a558e09dc5ee07fbe8182060aa40e43d2ed1d6055280007cbc372a6d7f1a56
+    name: andi
+    urls:
+    - andi-1.0.0.tgz
+    version: 1.0.0
```
Comment (ManApart) [inline docs/index.yaml]: Do we want to inc this version?

File: andi/values.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+replicaCount: 1
+
+image:
+  repository: smartcitiesdata/andi
+  tag: 0.2.3
```
Comment (ManApart) [inline andi/values.yaml]: This should be development here and then overridden with a specific tag in our deploy repo.

File: andi/values.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+ingress:
+  enabled: false
+  scheme: ""
+  subnets: ""
+  securityGroups: ""
+  certificateARN: ""
+  dnsZone: "localhost"
+
+resources:
```
Comment (ManApart) [inline andi/values.yaml]: This hasn't been followed as strictly, but we should really only have these resources in our overrides (deploy)  files, and just let the defaults be defaults in our charts-default.

Comment (LtChae) [reply andi/values.yaml]: Can you say defaults a few more times?

But seriously, what are you trying to convey?

File: docs/index.yaml (hunk 2 above)
Comment (LtChae) [reply docs/index.yaml]: This is the first version of this chart so far as the outside world is concerned. Version 1.0 seems appropriate.

File: andi/values.yaml (hunk 3 above)
Comment (LtChae) [reply andi/values.yaml]: Fixed

**Pull request merged**